src/utils/EventManager.ts

```
@@ -75,16 +75,24 @@ export const eventManager: EventManager = {
 		const existingListener = existingListeners.find((listener) => listener.callback === callback);
 		// If the listener hasn't been added, add it
 		if (!existingListener) {
-			const listenerInfo: EventListenerInfo<keyof HTMLElementEventMap> = {
+			const listenerInfo: EventListenerInfo<typeof eventName> = {
 				callback,
 				eventName,
 				target
 			};
+			// @ts-expect-error see: https://github.com/microsoft/TypeScript/issues/27808
```

ESLint

[error] Insert `␍`

(prettier/prettier)